src/Question.js

```diff
@@ -10,19 +10,32 @@ class Question extends Component {
       option: PropTypes.string.isRequired,
       correct: PropTypes.bool.isRequired
     })),
-    onClick: PropTypes.func.isRequired
+    onClick: PropTypes.func.isRequired,
+    questionNumber: PropTypes.number.isRequired,
+    userHasAnswered: PropTypes.bool.isRequired
+  }
+  getQuestionClassName() {
+    const questionClassName = {
+      correct ? 'answered' : 'wrong-answer'
+    }
+    if (this.props.userHasAnswered) {
+      return {questionClassName};
+    } else {
+      return 'question';
+    }
   }
   render() {
     return (
-      <div className="question">
+      <div className={this.getQuestionClassName} >
```

Comment:
I assume you mean put {this.getQuestionClassName()} ? 

Why so wordy? You could have just said add parentheses